apps/meeting/server.js

```diff
@@ -15,18 +15,29 @@ let protocol = 'http';
 let options = {};
 
 const chime = new AWS.Chime({ region: 'us-east-1' });
-const alternateEndpoint = process.env.ENDPOINT;
-if (alternateEndpoint) {
-  console.log('Using endpoint: ' + alternateEndpoint);
-  chime.createMeeting({ ClientRequestToken: uuid() }, () => {});
-  AWS.NodeHttpClient.sslAgent.options.rejectUnauthorized = false;
-  chime.endpoint = new AWS.Endpoint(alternateEndpoint);
+// Create ans AWS SDK Chime object. Region 'us-east-1' is globally available.
+const chimeRegional = new AWS.ChimeSDKMeetings({ region: 'us-east-1' });
+const chimeRegionalEndpoint = process.env.REGIONAL_ENDPOINT || 'https://meetings-chime.us-east-1.amazonaws.com';
+
+if (chimeRegionalEndpoint) {
+  console.info('Using regional endpoint', chimeRegionalEndpoint);
+  chimeRegional.endpoint = new AWS.Endpoint(chimeRegionalEndpoint);
 } else {
+  console.log('Using global endpoint ');
```

Comment:
Could we only use `console.info` or `console.log`?

Reply:
done